app/views/content/is-teaching-right-for-me/get-school-experience.md

```diff
@@ -42,24 +42,28 @@ During your experience, you’ll get to do things like:
 - observe lessons
 - see how teachers manage a classroom
 - find out how specific subjects are taught
-- speak to teachers and interact with pupils
+- speak to teachers and meet pupils
 - learn more about teacher training - including the application and interview process
 
 ## Find school experience
 
-You can search for and request [school experience](https://schoolexperience.education.gov.uk/) in England.
+If you enter your postcode into our Get school experience service you can find schools in your area that offer placements. You can filter these by the subject and age group you're looking to teach. It also tells you whether the type of experience available is in-school, virtual or both.
```

Comment:
```suggestion
Use our Get school experience service to search for and request experience in England.

If you enter your postcode, you can find schools in your area that offer placements. You can filter these by the subject and age group you're looking to teach. It also tells you whether the type of experience available is in-school, virtual or both.
```

Would it be worth introducing the service before talking about the filters? But otherwise I think this section makes much more sense now!